test/fixtures/default/sample.sh

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## These values you *MUST* motify to match your environment
+export TF_VAR_org_id="000000000000"
+export TF_VAR_billing_account="000000-000000-000000"
```

Comment:
Provide trivial way of setting this value, eg:

```
billing_account=$(gcloud alpha billing accounts list)

Reply:
I'd prefer we be explicit here and leave as is, especially when considering CI environments.

Reply:
Agreed that this should be explicit. My thinking is that we should provide examples for how to look up these values to make it easier to run tests. When a new contributor clones down the module and is determining how to run the tests we should guide them through setting up variables by providing example commands - but we shouldn't run them by default.